pallets/stateful-storage/src/stateful_child_tree.rs

```diff
@@ -377,9 +377,9 @@ impl<H: MultipartKeyStorageHasher> StatefulChildTree<H> {
 		storage_name: &[u8],
 	) -> ChildInfo {
 		let trie_root = Self::get_tree_prefix(msa_id);
-		// child tree root should be hashed by Twox128 to avoid probability of conflict
+		// child tree root should be hashed by Blake128 to avoid probability of conflict
```

Comment:
changing to Blake to use cryptographic hash

Reply:
Do we really need it? Is it worth the expense?

Reply:
From my perspective not but Parity recommended to use a Cryptographic hash function for child tree root. I don't think the expense is that bad compared to overall weights.